app/src/plotter.py

```diff
@@ -257,52 +271,91 @@ def fit_data(self):
-        while isinstance(x_range, dict):
+        while type(x_range) == dict:
             x_range = x_range[next(iter(x_range))]
 
         for code in self.config["plot_codes"]:
+
             # Create a Bokeh figure
-            max_value = self.get_max_value(code, self.config["filters_defaults"]["single_choice"])
+            max_value = self.get_max_value(
+                code, self.config["filters_defaults"]["single_choice"])
             self.plots[code] = figure(**self.config["general"],
-                                     x_range=[x_range[0], x_range[-1]],
-                                     y_range=[0, max_value])
+                                      x_range=[x_range[0] - .25,
+                                               x_range[-1] + .25],
+                                      y_range=[0, max_value])
 
             # Stretch to full width.
             self.plots[code].sizing_mode = "scale_width"
             self.plots[code].width_policy = "max"
-            
-            # Configure labels in x and y axes
+
+            # Change background color
+            self.plots[code].background_fill_color = self.config["background_fill_color"]
+            self.plots[code].border_fill_color = self.config["background_fill_color"]
+
+            # Hide x grid
+            self.plots[code].xgrid.grid_line_color = None
+
+            # Configure axes
             self.plots[code].xaxis.ticker = x_range
-            self.plots[code].yaxis.formatter.use_scientific = False
+            self.plots[code].yaxis.ticker.desired_num_ticks = 8
+            self.plots[code].yaxis.minor_tick_line_color = None
+            self.plots[code].yaxis.major_tick_line_color = None
+            self.plots[code].xaxis.major_tick_line_color = None
+
+            for axis in [self.plots[code].xaxis, self.plots[code].yaxis]:
+                axis.axis_label_text_font = self.config["text"]["font"]
+                axis.axis_label_text_font_style = self.config["text"]["font_style"]
+                axis.major_label_text_font = self.config["text"]["font"]
+                axis.major_label_text_font_style = self.config["text"]["font_style"]
+                axis.axis_line_width = self.config["axis"]["axis_line_width"]
+                axis.axis_line_color = self.config["background_fill_color"]
 
             # Add lines to the plot
-            colors = Category20[20]
+            colors = overwrite_colors
             for i, line_name in enumerate(self.config["filters"]["multi_choice"]):
-                self.plots[code].line(x="x", y=line_name, source=self.fitted_data[code], color=colors[i], legend_label=line_name)
+                self.plots[code].line(
+                    x="x", y=line_name, source=self.fitted_data[code], color=colors[i], line_width=4)
+                self.plots[code].circle(
+                    x="x", y=line_name, source=self.fitted_data[code], color=colors[i], size=16)
+                self.plots[code].circle(
```

Comment:
Does this line overwrite the previous one? If so, we can delete the previous one.

Reply:
Ah, got it 👍